components/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import '@fontsource/roboto';
```

Comment:
why you import this here ?

Reply:
Typography docs say that import '@fontsource/roboto' is necessary to use the Roboto font.
https://mui.com/material-ui/customization/typography/

To avoid import duplication, I've moved the import statement to main in 037002d.